awx/ui_next/src/screens/WorkflowApproval/WorkflowApprovalList/WorkflowApprovalList.test.jsx

```diff
@@ -0,0 +1,317 @@
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+import { WorkflowApprovalsAPI } from '../../../api';
+import { mountWithContexts } from '../../../../testUtils/enzymeHelpers';
+import WorkflowApprovalList from './WorkflowApprovalList';
+
+jest.mock('../../../api');
+
+const mockWorkflowApprovals = [
```

Comment:
Might consider putting this inside of a data.json file to reduce the # of lines of code in this file

Reply:
👍